endoreg_db/views/patient_examination/patient_examination.py

```
@@ -107,4 +121,20 @@ def update(self, request, *args, **kwargs):
                     {'error': f'Fehler beim Aktualisieren der Untersuchung: {str(e)}'}, 
                     status=status.HTTP_400_BAD_REQUEST
                 )
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    
+    def get_findings_for_examination(self, request, pk=None):
```

Ruff (0.12.2)

Unused method argument: `request`

(ARG002)